src/components/UI/Buttons/Buttons.tsx

```diff
@@ -1,19 +1,66 @@
 import { JSX } from 'preact';
 import styles from './Buttons.module.sass';
+import backIcon from './icons/backArrow.svg';
+import nextIcon from './icons/nextArrow.svg';
 
 interface ButtonProps extends JSX.HTMLAttributes<HTMLButtonElement> {
   onClick: () => void;
   children: JSX.Element | string;
```

Comment:
Add an option to assign multiple children elements.
```suggestion
  children: JSX.Element | JSX.Element[] | string;
```